kpet/run.py

```diff
@@ -50,6 +52,28 @@ def print_test_cases(patches, dbdir, pw_cookie=None):
         print(test_case)
 
 
+def pretty_xml(xml_content):
```

Comment:
I'd prefer this was at least named `maybe_pretty_xml()`, `try_prettify_xml()`, or something like that, or, better, that kpet would have an option to turn this off (or on), and the function doing it unconditionally. Otherwise kpet users as well as developers would find it confusing that this sometimes works and sometimes doesn't without apparent reason.

Another way could be requiring xmllint, but we can't do that with just Python packaging, as you know.